src/aggregation/metric/stats.rs

```diff
@@ -80,6 +129,110 @@ impl Stats {
     }
 }
 
+/// Extended stats contains a collection of statistics
+/// they extends stats adding variance, standard deviation
+/// and bound informations
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct ExtendedStats {
```

Comment:
Can you move the extended stats stuff to `extended_stats.rs`?

Reply:
Yes, but I need to change `IntermediateStats` visibility members and his `collect` method. if it's ok, I can refactor, do you prefer me adding accessor methods o declaring the fields as `pub`?

Reply:
you can add `pub(crate)`